deep_speech_2/train.py

```diff
@@ -78,6 +79,11 @@
     default='data/eng_vocab.txt',
     type=str,
     help="Vocabulary filepath. (default: %(default)s)")
+parser.add_argument(
+    "--init_model_path",
+    default='models/params.tar.gz',
+    type=str,
+    help="Model path for initialization. (default: %(default)s)")
```

Comment:
Add e.g. "If set None, the training will start from scratch. Otherwise, the training will resume from the existing model of this path".